dashboard/app/assets/javascripts/angular/testExecutor.service.js

```diff
@@ -25,7 +25,7 @@
           method: 'DELETE'
         }
       });
-      var testExecUpdateIntervalPeriod = 5; // seconds
+      var testExecUpdateIntervalPeriod = 1; // seconds
```

Comment:
@redconfetti it requests update from the dashboard back-end. Are you saying the frequency is too high or too low?